Portfolio/index.html

```diff
@@ -7,7 +7,34 @@
   </head>
```

Comment:
It seems that the HTML heading tag should be 'h1' instead of 'h3' to represent the main heading.